core/common/src/LocalTime.kt

```diff
@@ -197,6 +197,18 @@ public expect class LocalTime : Comparable<LocalTime> {
      * [kotlinx.datetime.format.DateTimeFormat] for [LocalTime] values.
      */
     public object Formats {
+        /**
+         * ISO 8601 basic format.
+         *
+         * Examples: `T1234`, `T123456`, `T123456.789`, `T123456.1234`.
+         *
+         * When formatting, seconds are always included, even if they are zero.
+         * Fractional parts of the second are included if non-zero.
+         *
```

Comment:
I think `LocalDateTime.ISO_BASIC` is also worth mentioning (given that you personally were interested in `LocalDateTime.Formats.ISO_BASIC`, for example).